assets/css/glimesh/components/chat.scss

```diff
@@ -89,4 +89,18 @@
     .chat-form {
         flex: 0 0;
     }
+}
+
+.chat-space-fill {
```

Comment:
I played around with this PR for awhile, and I noticed that removing this .chat-space-fill and the associated image in `lib/glimesh_web/live/user_live/stream.html.leex` have no effect on any browser size that I tested.

Tested:
1920×1080
1366×768
1440×900
1536×864
2560×1440
1680×1050
1280×720
1280×800
360×640
1600×900

What exactly is the selector for?

Reply:
To clarify, even when I removed `.chat-space-fill` the code that was changed in `stream.scss` was **perfect** on all of those resolutions.

Reply:
its to have something take up the space under the chat as it is somewhat floating and this would be only seen on laptop screens as these are the only one that has the right width and height ratio to activate this css styling ref in the picture
![glim](https://screenshots.wildwolf.dev/3395f5c213.png)

Reply:
Can you provide an example of a resolution I can test this on?

Reply:
1440x732

Reply:
this is the most common resolution for web browsers on large laptop screens with the favorites bar enabled

Reply:
Okay, I confirmed on that resolution that this `.chat-space-fill` is not requried for us to get the functionality of the PR.

.chat-space-fill existing
![image](https://user-images.githubusercontent.com/226638/108500081-44761580-727d-11eb-9032-773042f25ad2.png)


.chat-space-fill completely deleted
![image](https://user-images.githubusercontent.com/226638/108499997-25778380-727d-11eb-9538-96decd4591a6.png)

Can we remove this selector and the img?

Reply:
Can do that was more to have something there.

Reply:
I feel like we're just going to get people reporting that there's a glimdrop there :P

Reply:
XD